packages/node/src/integrations/spotlight.ts

```diff
@@ -0,0 +1,112 @@
+import type { Client, Integration } from '@sentry/types';
+import { logger, serializeEnvelope } from '@sentry/utils';
+import * as http from 'http';
+import { URL } from 'url';
+
+type SpotlightConnectionOptions = {
+  /**
+   * Set this if the Spotlight Sidecar is not running on localhost:8969
+   * By default, the Url is set to http://localhost:8969
+   */
+  sidecarUrl?: string;
+};
+
+/**
+ * Use this integration to send errors and transactions to Spotlight.
+ *
+ * Learn more about spotlight at https://spotlightjs.com
+ *
+ * Important: This integration only works with Node 18 or newer
+ *
+ * @param options
+ * @returns
+ */
+export class Spotlight implements Integration {
+  public name = 'Spotlight';
```

Comment:
we should set static id as well, shouldn't we?

Reply:
Good point, added it